regen-network/regen-web
# feat: APP-444 Navigate to Regen marketplace app for certain internal links from Terrasos app

## Description

Closes: https://regennetwork.atlassian.net/browse/APP-444



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



Check out the links on https://deploy-preview-2531--regen-marketplace.netlify.app/project/C45-010 go to dev.app.regen.network in a new tab:
- credit class card
- admin
- credit class, batch denom and issuer in Credit Issuance table

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
